src/components/general-components/general-component-style/map-card.less

```diff
@@ -0,0 +1,10 @@
+.card-header {
+  color: rgba(0, 0, 0, 0.45);
```

Comment:
🌱 We should try to use global colors for things like this

Reply:
Agreed

Reply:
That's part of the new ticket right?